2/simple_task_c++/plot.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+f = open('file.txt', 'r')
```

Comment:
I suggest using np.loadtxt, np.fromfile or pandas.readcsv helpers whenever possible as in more complex cases it is inefficient to write your own formatted file parser again and again.